docs/deploy/sso.md

```diff
@@ -0,0 +1,185 @@
+
+
+# Deploying Single Sign On
+Browsertrix allows for Single Sign On using either OIDC protocol or based on header values.
+
+Although it is technically possible to enable both OIDC and Header based SSO at the same time it is not suggested, to keep the user experience seamless.
+
+Single Logout is not supported as of now.
```

Comment:
Google has a good page on "[timeless documentation](https://developers.google.com/style/timeless-documentation)", have been following that, but admitidly this isn't in the style guide yet :)
```suggestion
Single Logout is not supported.
```